rpcclient/local.go

```diff
@@ -293,8 +293,33 @@ func (l *Local) TxSearch(ctx context.Context, query string, prove bool, page, pe
 }
 
 func (l *Local) Status(ctx context.Context) (*ctypes.ResultStatus, error) {
-	// needs block store, P2P layer
-	panic("Status - not implemented!")
+	latest, err := l.node.BlockStore.LoadBlock(l.node.BlockStore.Height())
+	if err != nil {
+		// TODO(tzdybal): extract error
```

Comment:
What does that mean?

Reply:
It's the reminder to extract/introduce sentinel error value.